fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerSimpleTest.java

```diff
@@ -377,92 +373,6 @@ public void readableAtEnd() {
         }
     }
 
-    @Test
-    public void run() {
-        Index index = runAsyncSetup();
-        try (OnlineIndexer indexBuilder = newIndexerBuilder()
-                .setIndex(index)
-                .setLimit(100).setMaxRetries(3).setRecordsPerSecond(10000)
-                .setMdcContext(ImmutableMap.of("mdcKey", "my cool mdc value"))
-                .setMaxAttempts(2)
-                .build()) {
-
-            AtomicInteger attempts = new AtomicInteger();
-
-            // Non-FDB error
-            attempts.set(0);
-            runAndHandleLessenWorkCodes(indexBuilder, store -> {
-                attempts.incrementAndGet();
-                throw new IllegalStateException("illegal state");
```

Comment:
It would be good to add back the tests of what happens for different error codes, and whether it lessens the limit, or retries or just fails. That might be easier if `Booker` is in change of that logic.

Reply:
✅